lib/list/ensureTextIndex.js

```diff
@@ -89,7 +89,9 @@ function ensureTextIndex (callback) {
 			}
 
 			// It's a text index but not one of ours; nothing we can do
-			debug('Existing text index \'' + existingIndexName + '\' in \'' + list.key + '\' not recognised; no action taken');
+			console.error('list.ensureTextIndex() failed to update the existing text index \'' + existingIndexName + '\' for the \'' + list.key + '\' list.');
```

Comment:
It should print very rarely but when it does happen it's pretty important to flag. Besides, it's not actually exiting or anything drastic like that..